lib/Mojo/IOLoop/Client.pm

```diff
@@ -171,14 +171,16 @@ sub _try_tls {
   my %options = (
     SSL_ca_file => $args->{tls_ca}
       && -T $args->{tls_ca} ? $args->{tls_ca} : undef,
+    SSL_ca_path => $args->{tls_ca_path}
+      && -d $args->{tls_ca_path} ? $args->{tls_ca_path} : undef,
     SSL_cert_file  => $args->{tls_cert},
     SSL_error_trap => sub { $self->emit(error => $_[1]) },
     SSL_hostname   => IO::Socket::SSL->can_client_sni ? $args->{address} : '',
     SSL_key_file   => $args->{tls_key},
     SSL_startHandshake  => 0,
-    SSL_verify_mode     => $args->{tls_ca} ? 0x01 : 0x00,
+    SSL_verify_mode     => ($args->{tls_ca} || $args->{tls_ca_path}) ? 0x01 : 0x00,
     SSL_verifycn_name   => $args->{address},
-    SSL_verifycn_scheme => $args->{tls_ca} ? 'http' : undef
+    SSL_verifycn_scheme => ($args->{tls_ca} || $args->{tls_ca_path}) ? 'http' : undef
```

Comment:
Does it make sense to do the ```-d``` test above, when SSL_verify_mode and SSL_verifycn_name does not?